patches/openbsd_gcc_patches/patch-gcc_gcc_config_opt

```diff
@@ -0,0 +1,10 @@
+--- gcc/config.gcc.orig	Thu Feb 11 14:26:19 2016
```

Comment:
The patches without `$OpenBSD$` didn't appear in OpenBSD CVS. The patches I have devised are only the hunks relating to position independent code (PIC and PIE).